pangeo-notebook/conda-lock.yml

```diff
@@ -41859,63 +41988,63 @@ package:
   category: main
   optional: false
 - name: xarray
-  version: 2024.7.0
+  version: 2024.2.0
   manager: conda
   platform: linux-64
   dependencies:
-    numpy: '>=1.23'
-    packaging: '>=23.1'
-    pandas: '>=2.0'
+    numpy: '>=1.23,<2.0a0'
+    packaging: '>=22'
+    pandas: '>=1.5'
     python: '>=3.9'
-  url: https://conda.anaconda.org/conda-forge/noarch/xarray-2024.7.0-pyhd8ed1ab_0.conda
+  url: https://conda.anaconda.org/conda-forge/noarch/xarray-2024.2.0-pyhd8ed1ab_0.conda
   hash:
-    md5: a7d4ff4bf1502eaba3fbbaeba66969ec
-    sha256: c8a0c70bb3402b29a9eebb1e41c5d28e9215bb14abea0c986d4d89026aa1ce42
+    md5: 8e25aab3323476d4fd0b5f6bad05d403
+    sha256: 4b0a8143f5c501246214823fe543e9d0749c950fdbdd39de6d8cd6209da2259f
   category: main
   optional: false
 - name: xarray
-  version: 2024.7.0
+  version: 2024.2.0
```

Comment:
Oh wait, reminder that we haven't fixed the xarray downgrade issue mentioned at https://github.com/pangeo-data/pangeo-docker-images/pull/577#discussion_r1744592203